PICMI_Python/fields.py

```diff
@@ -15,6 +15,13 @@ class PICMI_ElectromagneticSolver(_ClassWithInit):
     Electromagnetic field solver
       - grid: grid object to be used by the solver (grid object)
       - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', 'GPSTD' or 'ECT' (string)
+        - 'Yee': standard solver using the staggered Yee grid
```

Comment:
Part of #56

Reply:
Oops - my mistake. I started this PR from the branch of #56. I'll start over and submit a new PR from a fresh branch.